freqtrade/freqtradebot.py

```diff
@@ -359,11 +361,12 @@ def create_trade(self) -> bool:
         )
 
         # Create trade entity and return
-        self.rpc.send_msg(
-            f"""*{exc_name}:* Buying [{pair_s}]({pair_url}) \
-with limit `{buy_limit:.8f} ({stake_amount:.6f} \
-{stake_currency}, {stake_amount_fiat:.3f} {fiat_currency})`"""
-        )
+        self.rpc.send_msg({
+            'status':
+                f"""*{exc_name}:* Buying [{pair_s}]({pair_url}) \
```

Comment:
I think this should be abstracted further (into a dict with pair/url/limt/amount/...) and the compilation/formatting of the message should be left to the rpc handler ...

It augments the methods each handler needs to implement (i think by 2 - one for buy, one for sell) - however allows for more flexibility.

Reply:
@xmatthias Yes, +1 on that it should be abstracted further. I didn't thought about that while cherry-picking, I can change that within this PR if you want.

Reply:
that would be great - the PR i have in mind should concentrate on the "new" RPC handler as much as possible (ideally not have to touch `telegram.py` ... )